src/scripts/party/evolutions/Evolution.ts

```diff
@@ -41,8 +42,22 @@ abstract class Evolution {
 
         App.game.party.gainPokemonById(PokemonHelper.getPokemonByName(evolvedPokemon).id, shiny, true);
 
-        // EVs
         const evolvedPartyPokemon = App.game.party.getPokemonByName(evolvedPokemon);
+        if (newPokemon && App.game.challenges.list.realEvolutions.active()) {
+            const basePartyPokemon = App.game.party.getPokemon(PokemonHelper.getPokemonByName(this.basePokemon).id);
+            evolvedPartyPokemon.effortPoints = basePartyPokemon.effortPoints;
+            evolvedPartyPokemon.pokerus = basePartyPokemon.pokerus;
+            evolvedPartyPokemon.shiny = evolvedPartyPokemon.shiny || basePartyPokemon.shiny;
+            evolvedPartyPokemon.attackBonusAmount = basePartyPokemon.attackBonusAmount;
+            evolvedPartyPokemon.attackBonusPercent = basePartyPokemon.attackBonusPercent;
+            evolvedPartyPokemon.proteinsUsed = basePartyPokemon.proteinsUsed;
```

Comment:
Makes me nervous we may miss something if a new property gets added, but I'm not sure of a better solution.
RIP basePartyPokemon.heldItem

Reply:
Now moving held items. I have added a comment in party, so coders are aware of it